pkg/minikube/extract/extract.go

```diff
@@ -542,7 +540,25 @@ func extractAdvice(f ast.Node, e *state) error {
 
 		if i.Name == "Advice" {
 			// At this point we know the value in the kvp is guaranteed to be a string
-			advice, _ := kvp.Value.(*ast.BasicLit)
+			advice, ok := kvp.Value.(*ast.BasicLit)
+			if !ok {
+				// Maybe the advice is a function call, like fmt.Sprintf
+				ad, ok := kvp.Value.(*ast.CallExpr)
+				if !ok {
+					// Ok, we tried. Abort.
+					return true
+				}
+				// Let's support fmt.Sprintf with a string argument only
+				adArg, ok := ad.Args[0].(*ast.BasicLit)
```

Comment:
I don't know if it's ever possible that there are no `Args` and that `Args[0]` can cause an out of bounds panic, just mentioning it incase it is possible.

Reply:
good call, i'll just iterate through the args instead